examples/inference-deployments/mpt/mpt_7b_ft_handler.py

```diff
@@ -106,6 +112,7 @@ def __init__(self,
 
         self.end_id = end_id
 
+        print('Initializing FasterTransformer')
```

Comment:
are these for debugging or do you intend to leave these in?

Reply:
I was going to leave these in. the steps between these prints sometimes take minutes, so it's useful to see these if you pull the deployment logs while waiting for the server to become ready.